float32[] joint_positions
---
bool success
string error_msg
